SwrveSDK/src/main/assets/youtubeapi.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<body style = "padding: 0; margin: 0;">
+<div style = "width: 100%; height: 100%;" id="player"></div>
+
+<script>
```

Comment:
Could we move this to an API that we can upgrade without any changes to the SDK? Or a file on an S3 bucket